m3-sys/cminstall/src/config-no-install/AMD64_LINUX

```diff
@@ -1,10 +1,14 @@
 readonly TARGET = "AMD64_LINUX" % code generation target
 readonly GNU_PLATFORM = "amd64-linux" % "cpu-os" string for GNU
 
-SYSTEM_CC = "gcc -g -m64 -fPIC" % C compiler
-SYSTEM_CXXC = "g++ -g -m64 -fPIC" % C++ compiler
+SYSTEM_CC = "gcc -gdwarf-2 -gstrict-dwarf -m64 -fPIC" % C compiler
+SYSTEM_CXXC = "g++ -gdwarf-2 -gstrict-dwarf -m64 -fPIC" % C++ compiler
 
 readonly SYSTEM_ASM = "as --64" % Assembler
 
+% PIE debugging requires gdb 7.1 or newer, even for C
+% see http://www.gnu.org/software/gdb/download/ANNOUNCEMENT
+readonly POSITION_INDEPENDENT_EXECUTABLE = "-no-pie"
```

Comment:
Please mention that gdb was fixed in year 2010 or such and m3gdb is significantly even older.

Reply:
 I do not use "gdb". I did not use "gdb".
I use _only_ m3gdb(s):
 -- m3gdb built from Github-master on Debian 8.11.1
 -- m3gdb built from Github-master on CentOs 7.9 x64
 -- m3gdb built by R.B.
 -- etc.